views/partials/layout.njk

```diff
@@ -5,7 +5,12 @@
 {% block head %}
 
 <!--[if !IE 8]><!-->
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+<script src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8/jquery-ui.min.js" type="text/javascript"></script>
```

Comment:
seem to be mixing jquery ui versions here.  You've grabbed the 1.8 vresion of ui, but using the 1.10 theme?  Latest is 1.12